events/bingo/bingoWinnerApp/bingoWinner_app.js

```diff
@@ -0,0 +1,181 @@
+/* eslint-disable no-magic-numbers */
```

Comment:
why?

Reply:
cuz i copied and pasted lazily :D removing